### PR TITLE
Add esm entry for packages that already have main

### PR DESCRIPTION
**Pull request by dr3**
Contributes to #557

**Overall change:** Adds ESM entry to all packages that previously had a 'main'

**Code changes:**

- Adds ESM entry to all packages that previously had a 'main'

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <! --prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add esm entry to package |
```
Comment (thekp) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]: just a little change here to 1.3.0

File: packages/components/psammead-story-promo-list/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 1.1.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add esm entry to package |
```
Comment (thekp) [inline packages/components/psammead-story-promo-list/CHANGELOG.md]: bump version to 1.2.0

**Review by thekp — CHANGES_REQUESTED** [review]
version bump and changelog update for `psammead-image` and `psammead-story-promo`

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
 <! --prettier-ignore -->
```
Comment (thekp) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]: this is also incorrect syntax. might aswell fix it here

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
 # Psammead Image Placeholder Changelog
 
-<! --prettier-ignore -->
+<!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```
Comment (thekp) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]: only just noticed this jumps from 1.0.5 to 1.2.0

**Review by jroebu14 — APPROVED** [review]

**Review by thekp — APPROVED** [review]
you just need to resolve the conflicts 🛠 🔨

File: packages/components/psammead-inline-link/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+<<<<<<< HEAD
```
Comment (pjlee11) [inline packages/components/psammead-inline-link/CHANGELOG.md]:
```suggestion

```

conflict clean up

File: packages/utilities/psammead-test-helpers/CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```
Comment (pjlee11) [inline packages/utilities/psammead-test-helpers/CHANGELOG.md]:
```suggestion
| 1.1.0   | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```

File: packages/containers/psammead-timestamp-container/CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.1.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```
Comment (pjlee11) [inline packages/containers/psammead-timestamp-container/CHANGELOG.md:6]:
```suggestion
| 2.1.0   | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```

File: packages/components/psammead-timestamp/CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.1.0 | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```
Comment (pjlee11) [inline packages/components/psammead-timestamp/CHANGELOG.md:6]:
```suggestion
| 2.1.0   | [PR#1233](https://github.com/bbc/psammead/pull/1233) Add ESM modules entry |
```

**Review by pjlee11 — APPROVED** [review]

**Comment by dr3** [comment]
No testing needed, will regression when injested.

**Pull request merged**